SlevomatCodingStandard/Sniffs/Namespaces/AbstractFullyQualifiedGlobalReference.php

```diff
@@ -75,6 +76,11 @@ public function process(File $phpcsFile, $openTagPointer): void
 			$name = $referencedName->getNameAsReferencedInFile();
 			$namePointer = $referencedName->getStartPointer();
 
+			// We are in the middle of group use declaration
+			if (TokenHelper::findNextLocal($phpcsFile, T_CLOSE_USE_GROUP, $namePointer) !== null) {
```

Comment:
I don't this is right. I think we should just ignore the complete file if there's any group use. 

We don't support anything about group uses, so it's dangerous to try to fix the file.

Reply:
Ok, done.